### PR TITLE
Corrected example code masked_language_modeling.md

### PR DESCRIPTION
**Pull request by thirsty-camel**
i experienced an error when following the tutorial, and updated the code as i think it should be

# What does this PR do?

Fixes # (issue)

## Before submitting
- [* ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/en/tasks/masked_language_modeling.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Here is a first preprocessing function to join the list of strings for each exam
 
 ```py
 >>> def preprocess_function(examples):
-...     return tokenizer([" ".join(x) for x in examples["answers.text"]])
+...     return tokenizer(" ".join(eli5["train"][0]["answers"]["text"]))
```
Comment (LysandreJik) [inline docs/source/en/tasks/masked_language_modeling.md:136]: This should still use the `examples` passed as input, no? It's the preprocess function that's used below with the `map` of the dataset

Comment (SireMartin) [reply docs/source/en/tasks/masked_language_modeling.md:136]: Yes, sure it has to be examples indeed. I pushed some test code
I think it should be return tokenizer(" ".join(examples["answers"]["text"])), to tokenize all the sentences as one string. If not, i am not understanding it. I thought it would be wrong to tokenize all separate letters of the sentences.
But with this change i am getting "typerror: list indices must be integers of slices, not str", so propabely not a correct pr
